Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN \
 
 COPY supervisord.conf /etc/supervisord.conf
 COPY condor/*.conf /etc/condor/config.d/
-COPY start.sh update-config update-secrets /
+COPY start.sh update-config update-secrets create-flocking-tokens /
 COPY fetch-crl.cron /etc/cron.d/fetch-crl
 
 CMD ["/bin/bash", "-x", "/start.sh"]
```

Comment:
Side note: you should just use `supervisord_startup.sh` and `/etc/osg/image-config.d/`

Reply:
I'll add that to the 'technical debt todo list'